src/transformers/models/paligemma/processing_paligemma.py

```diff
@@ -73,6 +76,31 @@ def build_string_from_input(prompt, bos_token, image_seq_len, image_token):
     return f"{image_token * image_seq_len}{bos_token}{prompt}\n"
 
 
+class PaliGemmaTextKwargs(TextKwargs):
+    suffix: Optional[Union[TextInput, PreTokenizedInput, List[TextInput], List[PreTokenizedInput]]]
+
+
+class PaliGemmaImagesKwargs(ImagesKwargs):
+    do_convert_rgb: Optional[bool]
+    do_thumbnail: Optional[bool]
+    do_align_long_axis: Optional[bool]
+    do_rescale: Optional[bool]
+
+
+class PaliGemmaProcessorKwargs(ProcessingKwargs, total=False):
+    text_kwargs: PaliGemmaTextKwargs
+    image_kwargs: PaliGemmaImagesKwargs
+    _defaults = {
+        "text_kwargs": {
+            "tokenize_newline_separately": True,
```

Comment:
Looks like `tokenize_newline_separately` is not use anywhere, and it is not a default `text_kwargs`, so it might be best to remove it entirely?